src/main/java/org/ibp/api/rest/study/StudyEntryResource.java

```diff
@@ -51,17 +54,30 @@
 
 	}
 
-	@ApiOperation(value = "Create germplasm entries in study based on the specified germplasm list",
-		notes = "Create germplasm entries in study based on the specified germplasm list")
+	@ApiOperation(value = "Create germplasm entries in study based on the specified germplasm ids",
+		notes = "Create germplasm entries in study based on the specified germplasm ids")
 	@RequestMapping(value = "/{cropname}/programs/{programUUID}/studies/{studyId}/entries/generation", method = RequestMethod.POST)
```

Comment:
@mderamos This one can be 
`PUT /{cropname}/programs/{programUUID}/studies/{studyId}/entries`
to follow the same estandar we have for variables